core/tasty.cabal

```diff
@@ -61,7 +61,7 @@ library
   build-depends:
     base                 >= 4.9  && < 5,
     stm                  >= 2.3  && < 2.6,
-    containers                      < 0.7,
+    containers           >= 0.5.8 && < 0.7,
```

Comment:
GHC 8.0 ships `containers-0.5.7` according to https://downloads.haskell.org/~ghc/8.0.2/docs/html/users_guide/8.0.1-notes.html#containers, so maybe this bump is a little too early?

The CI here does not catch this. Haskell CI would because it works with `constraints: containers installed`.

(GHC 8.2 then ships with 0.5.10.)